examples/snowflake/.saturn/saturn.json

```diff
@@ -0,0 +1,22 @@
+{
+    "image": "saturncloud/saturn-rapids:2021.02.22",
+    "jupyter": {
+        "size": "g4dnxlarge",
+        "disk_space": "10Gi",
+        "ssh_enabled": false
+    },
+    "dask_cluster": {
+        "n_workers": 3,
+        "nthreads": 1,
+        "scheduler_size": "medium",
+        "worker_size": "g4dnxlarge"
+    },
+    "environment_variables": {
+        "TAXI_DATABASE": "saturn_nyc_taxi",
+        "JUPYTERLAB_SATURN_EXTENSION_DISABLED": "True"
+    },
```

Comment:
There is a notion in examples-cpu and examples-gpu of `required_credentials`. It'd be good to have a list of those here.